content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
```

Comment:
`We can organize ...` - I'm not sure we need this sentence, and `organizing` project usually means using some specific structure (like https://github.com/drivendata/cookiecutter-data-science) l... at least that the way I would read this

Reply:
Removed that part in 4f56d8f4